djconnectwise/sync.py

```diff
@@ -259,6 +259,11 @@ def fetch_delete_by_id(self, instance_id, pre_delete_callback=None,
                 )
             )
 
+    def _is_instance_changed(self, instance):
+        m2m_changed = self.m2m_changed \
+            if hasattr(self, 'm2m_changed') else False
```

Comment:
It doesn't seem quite right to me to `m2m_changed` check in the `Synchronizer`'s `_is_instance_changed`s method. For now, only `TeamSynchronizer` uses this so it should be kept on the `TeamSynchronizer` class. Can you give `TeamSynchronizer` its own `_is_instance_changed` method?

If code in the parent is only used for a single child class then that code should be in the child class.